PendingReleaseNotes

```diff
@@ -34,6 +34,14 @@
 
 >=16.2.8
 --------
+
+* MON/MGR: Pools can now be created with `--bulk` flag. Any pools created with `bulk`
+  will use a profile of the `pg_autoscaler` that provides more performance from the start.
+  However, any pools created without the `--bulk` flag will remain using it's old behavior
```

Comment:
"Note that pools created without the `--bulk` flag will default to the prior behavior."